src/components/modals/Modal/modal.module.css

```diff
@@ -0,0 +1,55 @@
+.Modal {
+  position: fixed;
+  z-index: 500;
```

Comment:
this property should be the part of backdrop css

Reply:
is also included in backdrop css